apps/api/src/scraper/scrapeURL/engines/fire-engine/index.ts

```diff
@@ -244,6 +244,7 @@ export async function scrapeURLWithFireEngineChromeCDP(
     timeout, // TODO: better timeout logic
     disableSmartWaitCache: meta.internalOptions.disableSmartWaitCache,
     mobileProxy: meta.featureFlags.has("stealthProxy"),
+    ddAntibot: meta.featureFlags.has("stealthProxy") || meta.options.proxy === "auto",
```

Comment:
Pretty sure this is incorrect. The stealth feature flag is added on retry, so no || check for auto is needed